manifests/params.pp

```diff
@@ -791,4 +791,10 @@
   } else {
     $verify_command = '/usr/sbin/apachectl -t'
   }
+
+  if ($::operatingsystem == 'RedHat' and ($facts['operatingsystemmajrelease'] == '8') ) {
```

Comment:
Probably better to check the os family and if it's `>= 8`. That way you also apply it to CentOS and future versions.

Reply:
Good plan!  Updated to resolve this  :)